app.js

```diff
@@ -6,7 +6,7 @@ var cookieParser = require('cookie-parser');
 var bodyParser = require('body-parser');
 
 var routes = require('./routes/index');
-var books = require('./routes/books');
+var books = require('./routes/addBooks');
```

Comment:
I would keep the routes files named after the route we want to display in the url.  For example, the add route should look like `/books/add`.  Let's discuss after lunch.